docs/user/tutorials/README.md

```diff
@@ -1,3 +1,48 @@
-# Application Connectivity - Tutorials
+<span class="fd-avatar--thumbnail fd-avatar fd-avatar--40 fd-avatar--accent-color-10" style="background-image: url('https://avatars.githubusercontent.com/tobiscr')" role="img"></span> [Tobias Schuhmacher](https://github.com/tobiscr) - Product Owner of Team Framefrog, Kyma runtime
 
-Browse the tutorials for Application Connectivity to learn how to use it step-by-step in different scenarios.
+Last update: Nov 2024
+
+# Integrate external system with Kyma
+
+## Table of Contents
+
+- [Integrate external system with Kyma](#integrate-external-system-with-kyma)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+    - [Prerequisites](#prerequisites)
+  - [Initial Setup](#initial-setup)
+    - [Create a Kyma Runtime and enable the required Modules](#create-a-kyma-runtime-and-enable-the-required-modules)
+  - [Steps](#steps)
+
+
+## Introduction
+
+The document describes the steps for connecting an external system (e.g. HTTPBin) with a Kyma runtime.
+
+In this example, Kyma will send an authenticated requests to an external service.
+
+
+### Prerequisites
+
```

Comment:
Maybe let's move everything from the **Create a Kyma Runtime and enable the required Modules** subsection to the Prerequisites? This way, we can get rid of the whole Initial Setup section.

Reply:
Adjusted :)